docs/earth-config/earth-config.md

```diff
@@ -37,23 +46,65 @@ git:
 
 Specifies the location where build data is stored. The default location is `/var/cache/earthly`.
 
+### cache_size_mb
+
+Specifies the total size of the buildkit cache, in MB. The buildkit daemon will allocate disk space for this size. Size less than `1000` (1GB) is not recommended. The default size if this option is not set is `10000` (10GB).
+
+This setting is only used when the buildkit daemon is started (or restarted). In order to apply the setting immediately, issue the command
+
+```bash
+earth --buildkit-cache-size-mb <cache-size-mb> prune --reset
+```
+
+### no_loop_device
+
+When set to true, disables the use of a loop device for storing the cache. By default, Earthly uses a file mounted as a loop device, so that it can control the type of filesystem used for the cache, in order to ensure that overlayfs can be mounted on top of it. If you are already using a filesystem compatible with overlayfs, then you can disable the loop device.
+
+### buildkit_image
+
+Instructs earth to use an alternate image for buildkitd. The default image used is `earthly/buildkitd:<earth-version>`.
+
+
 ## Git configuration reference
 
-### site
+git configuration is split up into global confi options, or site-specific options.
+
+### global options
+
+The global git options 
+
+#### url_instead_of
+
+Rewrites git URLs of a certain pattern. Similar to [`git-config url.<base>.insteadOf`](https://git-scm.com/docs/git-config#Documentation/git-config.txt-urlltbasegtinsteadOf).
+Multiple values can be separated by commas. Format: `<base>=<instead-of>[,...]`.
+
+This setting allows rewriting all git URLs of the form `https://...` into `git@github.com:...`, or vice-versa.
```

Comment:
We should perhaps give a non-github example here, because for github we recommend using the auth type setting. Maybe this point is also worth mentioning.

Reply:
how about something a bit more exotic like this?
```
* `--git-url-instead-of='git@example.com:=https://example.com/'` forces use of SSH-based URLs rather than HTTPS
* `--git-url-instead-of='https://localmirror.example.com/=git@example.com:'` forces use of HTTPS-based local mirror for ssh-based example.com repositories
```

Reply:
I changed the github references to example.com

Reply:
Perfect